releasenotes/notes/qiskit-dependency-3f6b8d71cc4d2c31.yaml

```diff
@@ -0,0 +1,10 @@
+---
+upgrade:
+  - |
+    The dependency on ``qiskit-terra`` was replaced with a dependency on
+    ``qiskit``. This change follows the move in upstream Qiskit to rename
```

Comment:
Should the old and new version numbers be explicitly mentioned here too?

Reply:
Done but we need to keep this release note in mind if we merge #1288 and increase the minimum qiskit version.